interface-definitions/include/bgp/bgp-common-config.xml.i

```diff
@@ -280,7 +280,7 @@
 </node>
 <node name="listen">
   <properties>
-    <help>Listen for and accept BGP dynamic neighbors from range</help>
+    <help>BGP Dynamic Neighbors listen commands</help>

```

Comment:
My only nitpick is inconsistent capitalization of "dynamic neighbors" across the file.